api/rosetta/balance.go

```diff
@@ -80,41 +91,41 @@ func (d *Data) Balance(ctx echo.Context) error {
 	// => https://github.com/optakt/flow-dps/issues/164
 
 	err = d.config.Check(req.NetworkID)
-	var netErr failure.InvalidNetwork
-	if errors.As(err, &netErr) {
-		return echo.NewHTTPError(http.StatusUnprocessableEntity, InvalidNetwork(netErr))
+	var netErr errors.InvalidNetwork
+	if errortype.As(err, &netErr) {
```

Comment:
Big no no. This is so idiomatic, you can't change it. Also, again, big disagreement about the need to rename the `rosetta` package, but if it is renamed, it should present an improvement, not a deterioration, of how this code reads.